models/shifts/frozen_shift_combinations_key.py

```diff
@@ -0,0 +1,12 @@
+import datetime
+from dataclasses import dataclass
+
+from .shifts_types_enum import ShiftTypesEnum
+
+
+@dataclass(frozen=True)
+class FrozenShiftCombinationKey:
+    employee_id: int
+    start_date_of_shift: datetime.date
+    end_date_of_shift: datetime.date
```

Comment:
bad data modeling
test at most one shift at the same time (no overlaps)
23:00 -> 3:00
2:00 -> 4:00